docs/data.rst

```diff
@@ -79,7 +79,9 @@ Sub-daily Output
 * ``prec`` : Precipitation (mm/timestep)
 * ``shortwave`` : Shortwave radiation (W/m^2)
 * ``vapor_pressure`` : Vapor pressure (kPa)
+* ``air_pressure`` : Air pressure (kPa)
 * ``rel_humid`` : Relative humidity
+* ``spec_humid`` : Specific humidity
```

Comment:
Lets add units for these two
```
Relative humidity (%)
Specific humidity (g/g)
``` 

explicit is better than implicit (or so they say)